chapters/appendix-IV-SPDX-license-expressions.md

```diff
@@ -1,175 +1,174 @@
+Often a single license can be used to represent the licensing terms of a source code or binary file, but there are situations where a single license identifier is not sufficient. A common example is when software is offered under a choice of one or more licenses (e.g., `GPL-2.0 OR BSD-3-Clause`). Another example is when a set of licenses is needed to represent a binary program constructed by compiling and linking two (or more) different source files each governed by different licenses (e.g., `LGPL-2.1 AND BSD-3-Clause`).
 
-SPDX License Expressions provides a way for one to construct expressions that more accurately represent the licensing terms typically found in open source software source code. A license expression could be a single license identifier found on the SPDX License List; a user defined license reference denoted by the LicenseRef-{idString}; a license identifier combined with an SPDX exception; or some combination of license identifiers, license references and exceptions constructed using a small set of defined operators (e.g., AND, OR, WITH and +). We provide the definition of what constitutes a valid an SPDX License Expression in this section. 
+SPDX License Expressions allow accurate representations of the licensing terms typically found in open source software. A license expression could be a single license identifier found on the [SPDX License List](appendix-I-SPDX-license-list.md); a user defined license reference denoted by the `LicenseRef-{idString}`; a license identifier combined with an SPDX exception; or some combination of license identifiers, license references and exceptions constructed using a small set of defined operators (`AND`, `OR`, `WITH` and `+`).  We provide the definition of what constitutes a valid an SPDX License Expression in this section.
 
-The exact syntax of license expressions is described below in [ABNF](http://tools.ietf.org/html/rfc5234).
+## IV.2 Syntax <a name="IV.2"></a>
 
-idstring = 1*(ALPHA / DIGIT / "-" / "." )
+The exact syntax of license expressions is described below in [ABNF][rfc5234].  `ALPHA`, `DIGIT`, and `WSP` are from the [ABNF core rules][rfc5234-aB].
 
-license-id = \<short form license identifier in [Appendix I.1](./appendix-I-SPDX-license-list.md)>
+```
+idstring                    = 1*(ALPHA / DIGIT / "-" / "." )
+license-id                  = <short form license identifier>
+license-exception-id        = <short form license exception identifier>
+license-ref                 = ["DocumentRef-" idstring ":"] "LicenseRef-" idstring
+license                     = license-id / license-ref
+space                       = 1*WSP
+simple-expression           = license ["+"] [space "WITH" space license-exception-id]
+binary-compound-operator    = "AND" / "OR"
+license-expression          = simple-expression
+                            / license-expression space binary-license-operator space license-expression
```

Comment:
binary-license-operator -> binary-compound-operator

Reply:
> binary-license-operator -> binary-compound-operator

I'll go with `binary-expression-operator` because the arguments are `license-expression`s.

Reply:
Addressed in cf8decf → 9c8d189.